easybuild/easyconfigs/p/pasta/pasta-1.8.5-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'PythonPackage'
+
+name = 'pasta'
+version = '1.8.5'
+local_commit = '926a4eb'
+
+homepage = 'https://github.com/smirarab/pasta'
+description = """PASTA (Practical Alignment using SATe and Transitivity)"""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+github_account = 'smirarab'
+source_urls = [GITHUB_SOURCE, 'https://github.com/smirarab/sate-tools-linux/archive']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ},
+           {'download_filename': '90fb074.tar.gz', 'filename': 'sate-tools-linux-90fb074.tar.gz'}]
+
+checksums = [
+    'e809562011266e2ed13a5ca4cf0accf9d62093f6885852828e07f3ebca325ac5',  # pasta-1.8.5.tar.gz
+    '48be3308d8f4e3bf92df528a1d6b00ad422376bedee12d4dcb9c1840f1f34f17',  # sate-tools-linux-90fb074.tar.gz
+]
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+skipsteps = ['build']
+
+dependencies = [
+    ('DendroPy', '4.4.0'),
+]
+
+preinstallopts = 'mv %(builddir)s/sate-tools-linux* ../sate-tools-linux && '
+install_target = 'develop'
+use_pip = False
+download_dep_fail = True
+
+sanity_check_paths = {
+    'files': ['bin/run_pasta.py'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
```

Comment:
You should add a sanity check command like `run_pasta.py --help`, because the script doesn't actually work right now (even after fixing the shebang):

```
Traceback (most recent call last):
  File "/software/pasta/1.8.5-GCC-8.2.0-2.31.1/bin/run_pasta.py", line 4, in <module>
    __import__('pkg_resources').require('PASTA==1.8.5')
  File "/software/Python/3.7.2-GCCcore-8.2.0/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3191, in <module>
    @_call_aside
  File "/software/Python/3.7.2-GCCcore-8.2.0/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3175, in _call_aside
    f(*args, **kwargs)
  File "/software/Python/3.7.2-GCCcore-8.2.0/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3204, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/software/Python/3.7.2-GCCcore-8.2.0/lib/python3.7/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/software/Python/3.7.2-GCCcore-8.2.0/lib/python3.7/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/software/Python/3.7.2-GCCcore-8.2.0/lib/python3.7/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'PASTA==1.8.5' distribution was not found and is required by the application
```

It looks like `pasta` isn't actually being installed to `lib/python*/site-packages` in the `pasta` install dir?